### ISSUE TITLE
Generator is working incorrecly

### ISSUE THREAD

**Issue by AlexXanderGrib** [bcherny/json-schema-to-typescript#296]
Hello,  today i tried to generate typings for VK API using your tool.  (command line version `json2ts`)
  
Input: json schema files from this repo https://github.com/VKCOM/vk-api-schema (edited: removed pointers to undefined objects)  
  
Output (almost, exclude schema.json, looked like this):  
```TS
/* tslint:disable */
/**
 * This file was automatically generated by json-schema-to-typescript.
 * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
 * and run json-schema-to-typescript to regenerate this file.
 */

export interface Methods {
  [k: string]: unknown;
}
```

**Comment by bcherny** [comment]
Hey there! schema.json is the only file that have a top-level `properties` field. Without this field, jstt doesn't know what the relationships between definitions are.

One potential enhancement would be to generate types for definitions, even if they're not referenced by `properties`.

**Comment by bcherny** [comment]
Whoops, I forgot we have a flag for this.

If you do `json2ts --unreachableDefinitions`, does that work for you?

**Comment by AlexXanderGrib** [comment]
Thanks  for your help, by I don't anymore need json2ts, beaceuse of size and bullsh*t in this files I decided to write my own specific lightweight json2ts.

And again, thanks for your work and please optimise your tool somehow, because on large files, like this it crashes node

**Comment by Salzian** [comment]
To end this issue on a good note... your answer helped me @bcherny 

I was trying to transfer interface information from my backend application to my frontend React app. So I tried combining json-schema-to-typescript with [typescript-json-schema](https://www.npmjs.com/package/typescript-json-schema). Generate schema file from backend models and generate interfaces for the frontend.

That tool unfortunately wraps everything in a `"definitions"` block making it unreadable form json2ts :/ with the `--unreachableDefinitions` flag however it magically worked! I do however have a `interface Schemas` floating around in the file but whatever.

I then realised... I could just cp the interface from the backend to the frontend. It's a mono repo lul
